### PR TITLE
Feat: Home Added

### PR DESCRIPTION
**Pull request by Rohanmond**
Added index.html for my ecommerce webapp Fashify Store
File Structure:
doc_styles
        doc_home.css(having the CSS for home)
        doc_style.css(main file for adding the all CSS)
pages(components)
src(images)
index.html
style.css(main CSS file)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  <body>
+    <div class="home-container">
+      <nav class="navigation">
+        <div class="nav-left">
+          <div class="nav-logo-container">
+            <a href="">
+              <img
+                class="hero-logo"
+                src="./src/logo/Fashify Transparent pr.png"
+                alt=""
```
Comment (manish591) [inline index.html:20]: add alt for images.

File: doc_style/doc_home.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    width: 25rem;
+  }
+}
+@media screen and (max-width: 800px) {
+  .home-cards {
+    flex-direction: column;
+  }
+  .home-card-container {
+    flex-direction: column;
+  }
```
Comment (manish591) [inline doc_style/doc_home.css]: You can write this like
.home-cards, 
.home-cards-container {
  flex-direction: column;
  }

File: index.html (hunk 1 above)
Comment (Rohanmond) [reply index.html:20]: it is fixed now

File: doc_style/doc_home.css (hunk 2 above)
Comment (Rohanmond) [reply doc_style/doc_home.css]: It is also fixed now

**Review by anuj-kumary — APPROVED** [review]
Looks Good!!

**Pull request merged**